readthedocs/organizations/views/private.py

```diff
@@ -69,6 +71,37 @@ def get_queryset(self):
         return Organization.objects.for_user(user=self.request.user)
 
 
+class ChooseOrganization(ListOrganization):
+    template_name = "organizations/organization_choose.html"
+
+    def dispatch(self, request, *args, **kwargs):
+
+        # We're overwriting a mixin dispatch()
+        if not settings.RTD_ALLOW_ORGANIZATIONS:
+            raise Http404
```

Comment:
You should be able to override the `get` method instead of `dispatch`, that way you don't need to duplicate this.

Reply:
this is true :+1: fixed so we don't have duplication :+1: 

will add this to my lightning talk draft "decorators are friends of CBVs" -- i.e. not having to pick and choose between dispatch, get, get_context_data etc in order to add functionality when class inheritance is complicated :)

Reply:
yeah, I also don't like much using mixins, most of the time they can be just functions.